### PR TITLE
src: renaming SilentEcho* -> VirtualDevice*

### PR DESCRIPTION
**Pull request by chris-ramon**
#### Overview
- Renames:
  - [x] SilentEcho -> VirtualDevice
  - [x] SilentEchoScript -> VirtualDeviceScript
  - [x] SilentEchoValidator -> VirtualDeviceValidator
  - [x] SilentEcho* -> VirtualDevice*
- README: Changes SilentEcho* to VirtualDevice*.
- Closes: https://github.com/bespoken/silent-echo-sdk/issues/28

##### Before Merge
- [x] Publish to npm: https://www.npmjs.com/package/virtual-device-sdk
##### After Merge
- [ ] Rename the repo: `bespoken/silent-echo-sdk` -> `bespoken/virtual-device-sdk`.
- [ ] Reconfigure CircleCi & Codecov to use new repo name: `bespoken/virtual-device-sdk`.

#### Test plan
- Unit test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
 [![CircleCI](https://circleci.com/gh/bespoken/silent-echo-sdk.svg?style=svg)](https://circleci.com/gh/bespoken/silent-echo-sdk)
 [![codecov](https://codecov.io/gh/bespoken/silent-echo-sdk/branch/master/graph/badge.svg)](https://codecov.io/gh/bespoken/silent-echo-sdk)
-[![npm](https://img.shields.io/npm/v/silent-echo-sdk.svg)](https://www.npmjs.com/package/silent-echo-sdk)
+[![npm](https://img.shields.io/npm/v/virtual-device-sdk.svg)](https://www.npmjs.com/package/virtual-device-sdk)
 
-# Silent Echo SDK
-Use the Silent Echo SDK to build UIs and bots that interact with Alexa via text.
+# Virtual Device SDK
+Use the Virtual Device SDK to build UIs and bots that interact with Alexa via text.
```
Comment (jkelvie) [inline README.md]: Use the Virtual Alexa SDK to test Alexa without using our voice.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
 
-Check out our first example project to use it - [SilentEchoBot](https://github.com/bespoken/silent-echo-bot)!  
-Add SilentEcho to your Slack - [try it here](https://silentechobot.bespoken.io/slack_auth).
+Check out our first example project to use it - [VirtualDeviceBot](https://github.com/bespoken/silent-echo-bot)!
```
Comment (jkelvie) [inline README.md]: Delete these next two lines.

File: README.md [hunk 3]
```diff
@@ -56,23 +56,23 @@ export interface ICard {
 ```
 
 # HTTP SDK
-The SilentEcho service can also be called directly via HTTP.
+The VirtualDevice service can also be called directly via HTTP.
 
 ## Pre-Requisites
-Get a SilentEcho Token:  
-[https://silentecho.bespoken.io/link_account?token=true](https://silentecho.bespoken.io/link_account?token=true)
+Get a VirtualDevice Token:
```
Comment (jkelvie) [inline README.md]: They need to get this via the Dashboard URL now. This should say:
* Sign into the [Bespoken Dashboard](https://apps.bespoken.io/dashboard)
* Create a source
* Select the Validation tab and follow the instructions there

File: README.md (hunk 1 above)
Comment (chris-ramon) [reply README.md]: Do you mean `Virtual Device SDK` instead `Virtual Alexa SDK` ?

**Pull request merged**